include/util/coordinate_calculation.hpp

```diff
@@ -24,14 +25,14 @@ const constexpr double EARTH_RADIUS_WGS84 = 6378137.0;
 
 namespace detail
 {
-    // earth circumference devided by 2
-    const constexpr double MAXEXTENT = EARTH_RADIUS_WGS84 * boost::math::constants::pi<double>();
-    // ^ math functions are not constexpr since they have side-effects (setting errno) :(
-    const double MAX_LATITUDE = RAD_TO_DEGREE * (2.0 * std::atan(std::exp(180.0 * DEGREE_TO_RAD)) - boost::math::constants::half_pi<double>());
-    const constexpr double MAX_LONGITUDE = 180.0;
+// earth circumference devided by 2
+const constexpr double MAXEXTENT = EARTH_RADIUS_WGS84 * boost::math::constants::pi<double>();
+// ^ math functions are not constexpr since they have side-effects (setting errno) :(
+const double MAX_LATITUDE = RAD_TO_DEGREE * (2.0 * std::atan(std::exp(180.0 * DEGREE_TO_RAD)) -
+                                             boost::math::constants::half_pi<double>());
+const constexpr double MAX_LONGITUDE = 180.0;
```

Comment:
Didn't touch it. So formatting. The question remains how we can get a consistent formatting in...